web/src/components/typing/Typing.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+
+interface TypingProps {
+    staticText?: string;
+    typedWords: Required<string[]>;
+    intervalTime?: number;
+}
+const Typing = (props: TypingProps) => {

```

Comment:
use camelCase for prop interface name

please destruct these props 
{ something, something2 }